src/Components/test/testassets/BasicTestApp/RouterTest/Links.razor

```diff
@@ -20,7 +20,8 @@
     <li><NavLink href="/subdir/WithParameters/Name/Abc/LastName/McDef">With more parameters</NavLink></li>
     <li><NavLink href="/subdir/LongPage1">Long page 1</NavLink></li>
     <li><NavLink href="/subdir/LongPage2">Long page 2</NavLink></li>
-    <li><NavLink href="/subdir/WithLazyAssembly">With lazy assembly</NavLink></li>
+    <li><NavLink href="/subdir/WithLazyAssembly" id="with-lazy-assembly">With lazy assembly</NavLink></li>
+    <li><NavLink href="/subdir/WithLazyLoadedRoutes" id="with-lazy-routes">With lazy loaded routes</NavLink></li>
```

Comment:
Is this a new component?

Reply:
This component already exists. We were previously testing its ability to lazy-load in isolation though. 

We had missing test coverage in validating the behavior when you navigated from a route that lazily-loaded assemblies to another route that lazily loaded them. The test I added validates that. The use of links instead of `SetUrlViaPushState` isn't relevant here but I wanted to include a test that used the navigation technique users were most likely to have.